extensions/amp-story/1.0/amp-story-access.js

```diff
@@ -67,7 +86,15 @@ export class AmpStoryAccess extends AMP.BaseElement {
 
   /** @override */
   buildCallback() {
-    const drawerEl = renderAsElement(this.win.document, TEMPLATE);
+    const storyEl = closestByTag(this.element, 'AMP-STORY');
+    const logoSrc = storyEl && storyEl.getAttribute('publisher-logo-src');
+
+    if (!logoSrc) {
+      user().warn(
```

Comment:
Right. My point was that it's not required by this component, and `error()` would stop the code execution. Should we let users implement the feature even without being blocked by the logo, or do we want to really ensure it can't run if the logo is missing?